File: PHP/library/DataModels/CustomerOrder.php
<?php

namespace DataModels;

class CustomerOrder {
	/*
	 * @return int
	 */
	public $orderId;
	
	/*
	 *@return DateTime 
	 */
	public $orderDate;
	
	/*
	 * @return CustomerInfo
	 */
	//public $customer;
	
	/*
	 * @return array[];
	 */
	//public $items;
	
}

class ItemDetails{
	/*
	 * @return int
	 */
	public $quantity;
	
	/*
	 * @return float
	 */
	public $price;
	
	/*
	 * @return string
	 */
	public $name;
	
	/*
	 * @return string
	 */
	public $description;
	
}

class CustomerInfo{
	/*
	 * @return int
	 */
	public $customerId;
	
	/*
	 * @retun string
	 */
	public $name;
	
	/*
	 * @return string
	 */
	public $address;
	
	/*
	 * @return string
	 */
	public $zip;
}
?>

File: CSharp/FailSafeTDD_Example/src/app/DataModels/CustomerOrder.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataModels
{
    public class CustomerOrder
    {
        public int OrderId { get; set; }
        public DateTime OrderDate { get; set; }
        public CustomerInfo Customer { get; set; }
        public List<ItemDetails> Items { get; set; }

    }

    public class ItemDetails
    {
        public int Quantity { get; set; }
        public Decimal Price { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
    }

    public class CustomerInfo
    {
        public int CustomerId { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public int Zip { get; set; }

    }
}
